stubs/ldap3/ldap3/core/connection.pyi

```diff
@@ -1,8 +1,9 @@
 from _typeshed import Self
 from types import TracebackType
-from typing import Any
+from typing import Any, Iterable
```

Comment:
`Iterable` should be imported from `collections.abc`.

Reply:
Thank you. In general is `collections.abc` preferred over what's in the `typing` module?

Reply:
Yes, where possible. The typing names are deprecated now.